Automattic/pocket-casts-android
# Update skip icons for widgets

## Description

Update skip forward icon to be symmetrical with skip back icon.

## Testing Instructions

1. Add a medium or large widget to you home screen.
2. Verify that skip button icons are symmetrical.

## Screenshots or Screencast 

| Before | After |
| - | - |
| ![a](https://github.com/Automattic/pocket-casts-android/assets/30936061/37ef9e7e-de4f-4a0c-96f4-c75bcfaf050f) | ![b](https://github.com/Automattic/pocket-casts-android/assets/30936061/e6254e4b-0684-4f63-b90f-49732c5e712e) |


## Checklist
 
#### I have tested any UI changes...
